### DISCUSSION TITLE
Automated invoice email

### DISCUSSION THREAD

**Discussion by gcrickman** [ParabolInc/parabol#6569] [Feedback]
## Issue - Enhancement

[A Pro account](https://app.hubspot.com/contacts/3888472/deal/1217866492) 🔒   has requested that monthly invoices are automatically emailed to them after they are paid to satisfy accounting requirements. 

#### Users can:
- Set up auto invoice emails to a specific email address

#### Design:
- Iterated on [these designs](https://www.figma.com/file/UI703B8Hd7d7EleOyqXMxk/Invoice-Mailing-List?node-id=5%3A159) and drafted an implementation issue
 
### Estimated effort
- 10 hours

**Comment by avivapinchas** [comment]
Adding a ➕ here - we should send receipts when we charge people money.

**Comment by gcrickman** [comment]
This was requested by another [client](https://app.hubspot.com/contacts/3888472/deal/610983384) 🔒   in Germany

**Comment by jordanh** [comment]
Got another request for this here from [this customer](https://app.hubspot.com/contacts/3888472/contact/1430601) 🔒

**Comment by jordanh** [comment]
Updating this issue to be a design issue, as it'll need designs we can execute on before we can implement.

AC
Allowing billing leaders to:
   - Designate which email addresses to end emails to
      - Any email should be able to be added, it should not be a requirement that the email is associated with a user (JTBD: when I want to send a copy of the receipt to our accounting folks, I want to be able to enter their email address, so that they can receive it automatically)
      - Aside from Create, also delete these email entries
   - Send an email when a payment is successful
   - Send an email when a payment is unsuccessful

EE
   - 30 hours

**Comment by ackernaut** [comment]
Do this older design work? https://www.figma.com/file/UI703B8Hd7d7EleOyqXMxk/Invoice-Mailing-List?node-id=9%3A544

**Comment by jordanh** [comment]
Oh yeah, those are totally rad. I'm glad I asked!

**Comment by jordanh** [comment]
Updated AC and EE

[a comment by an automated account is not shown]

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/24634251) 🔒 writes:

> I don't get the Parabol invoices to my mailbox. 
> 
> It would be great to have this option as we're dealing with numerous services and it's easy to get lost. 
> 
> I looked for email preferences but couldn't find anything - any way you could help me with that?

**Comment by jordanh** [comment]
Removing stale. I know @gcrickman, @drewhousman1010, and the rest of the Sales crew gets asked this nearly once a week.

Adding `Batting Practice` to this as I think it would be a good way to evaluate folks skillz

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/9852851) 🔒  says:

> Given I'm about to go on maternity leave, I was wondering how best for us to ensure that our finance team get the invoices for our Parabol account. Would you be able to either switch out my email address for the one cc'd on the account, or if there are invoices emailed could you make a note to put our finance team on CC?

**Comment by ackernaut** [comment]
> Do this older design work? https://www.figma.com/file/UI703B8Hd7d7EleOyqXMxk/Invoice-Mailing-List?node-id=9%3A544

Here was an earlier design to address this. I propose we 1) iterate on that or 2) scrap it for A Better Idea™ and then write an implementation issue based on the design output.

**Comment by ackernaut** [comment]
AC and EE updated

**Comment by ackernaut** [comment]
> A [user](https://app.hubspot.com/contacts/3888472/contact/24634251) 🔒 writes:
> 
> > I don't get the Parabol invoices to my mailbox.
> > It would be great to have this option as we're dealing with numerous services and it's easy to get lost.
> > I looked for email preferences but couldn't find anything - any way you could help me with that?

I showed the admin prototype to this user. [At the 51:39 timestamp](https://www.loom.com/share/becad5a3855241659d7ef1c53b26f3f2), with the general question of how Parabol is working, they brought this up again. It would save them a lot of time having to go hunt invoices and submit them if they could just enter an email that would automatically receive them.

**Comment by gcrickman** [comment]
These are requirements for [this](https://app.hubspot.com/contacts/3888472/deal/5041553723) account 🔒 

Following things are required on a valid invoice, multiple of those are missing on the billing page:

Name and address of your company (service provider)
Our company name and address (being billed as customer / service consumer / customer)
Your company (service providers) tax number
Date of issuing of the invoice
Invoice number (unique / sequentially)
Quantity / Amount and of delivered goods or service
Date of (service) delivery, if not equal to Date of invoice
Price, including discounts (if not included in pricing), if applicable separated by different tax rates
included taxes (rate and amount included), or notice about tax exemption

**Comment by drewhousman1010** [comment]
This[ Pro account](https://app.hubspot.com/contacts/3888472/deal/5279161705) 🔒 also requires automated invoicing and has requested that we manually email them an invoice each month.

**Comment by drewhousman1010** [comment]
Got a request from this [user](https://app.hubspot.com/contacts/3888472/contact/9958351) 🔒 

> Hi I want to download invoices issued toward my company

If they were getting the invoices sent automatically it would probably make their life a lot simpler.

**Comment by gcrickman** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/11132901) 🔒  writes:

> Hi there, can I put the company address on the invoice?

**Comment by gcrickman** [comment]
> A [user](https://app.hubspot.com/contacts/3888472/contact/11132901) 🔒 writes:
> 
> > Hi there, can I put the company address on the invoice?

Now doing a monthly manual invoice for this customer.

**Comment by ackernaut** [comment]
I want to take on this design work in Sprint 101 on Squad 2. We’ll see!

[Taking feature notes here](https://www.notion.so/parabol/Automated-invoice-emails-95597e0a4e3a43aa9f4c59e18d101132) 🔒

**Comment by ackernaut** [comment]
We done a little [exploration about generating PDFs](https://github.com/ParabolInc/parabol/issues/2730#issuecomment-1104398129) for meeting summaries. We might be able to continue that here in the interest of generating invoice PDFs that we can attach to emails or download from the in-app billing view.

**Comment by gcrickman** [comment]
A [customer](https://app.hubspot.com/contacts/3888472/contact/29933401) 🔒  writes:

> hi, i am a billing leader within parabol and would like to get the invoice sent to me by email

**Comment by ackernaut** [comment]
See burn-down issue for automatic invoice emails #6574
